projects/core/tokens/first-day-of-week.ts

```diff
@@ -0,0 +1,10 @@
+import {InjectionToken} from '@angular/core';
+import {TuiDayOfWeek} from '@taiga-ui/cdk';
+
+export const TUI_FIRST_DAY_OF_WEEK = new InjectionToken<TuiDayOfWeek>(
+    'The first day of the week index',
+    {
+        providedIn: 'root',
```

Comment:
nit: you can remove this line, it's provided in root by default

Reply:
deleted